src/backend/feed/processor.js

```diff
@@ -147,7 +147,8 @@ module.exports = async function processor(job) {
           logger.warn(`${info.status} Feed no longer available: ${feed.url}`);
           break;
         case 429:
-          logger.warn(`${info.status} Feed requested too many times: ${feed.url}`);
+          logger.warn(`${info.status} Feed requested too many times, setting delay: ${feed.url}`);
+          feed.setDelayed(process.env.FEED_PROCESSING_DELAY_SEC || 3600);
```

Comment:
This returns a Promise, since it talks to Redis.  You need to `await feed.setDelayed...`

Reply:
Done: https://github.com/Seneca-CDOT/telescope/pull/726/commits/ab0ec50816580a3a5e72c4cb48c9d55afe6b0cfa